### DISCUSSION TITLE
Category Updation for Interactive Video

### DISCUSSION THREAD

**Discussion by Xitija** [Sunbird-Knowlg/Community#15] [Q&A] [answered]
Hello All,
We have created a new category for Interactive Video and we want it to be configured such that it is selected as a default category in while creating the Interactive Video (Video + Question set) in the portal .
We have the following approaches for doing it:

By updating the channel where the category is picked from for the Question Set
OR
By having a config such that we can have the category listed
Can you please help us in choosing which approach we should go with ?
@coolbung @baljeetsingh @vaivk369 @pallakartheekreddy

We tried following [https://knowlg.sunbird.org/learn/product-and-developer-guide/editors](https://knowlg.sunbird.org/learn/product-and-developer-guide/editors) but could not get much help. 

Attaching a screenshot of the form from https://dock.sunbirded.org/contribute where category is to be updated in the 'Type' form field. 
![categoryticket](https://user-images.githubusercontent.com/65657373/171356096-039577f4-391a-440e-9085-e70bd32dabc2.png)

**Comment by vinukumar-vs** [comment, marked as answer]
@pallakartheekreddy Suggest how can @Xitija create interactive video content. This is related to dock & question-set, I would suggest to move this to Sunbird-Inquiry.

@Xitija move this discussion to [Sunbird-Inquiry](https://github.com/Sunbird-inQuiry/Community/discussions)

@Krishnaj20

**Reply by Xitija** [reply]
Ok moved it [here](https://github.com/Sunbird-inQuiry/Community/discussions/15)
